src/EditorFeatures/Test/BraceHighlighting/AbstractBraceHighlightingTests.cs

```diff
@@ -16,9 +17,9 @@ namespace Microsoft.CodeAnalysis.Editor.UnitTests.BraceHighlighting
 {
     public abstract class AbstractBraceHighlightingTests
     {
-        protected async Task TestBraceHighlightingAsync(string markup)
+        protected async Task TestBraceHighlightingAsync(string markup, CSharpParseOptions options = null)
```

Comment:
Hmm, can you please check if `AbstractBraceHighlightingTests` is the base class for C# and VB brace highlighting tests? If so, then, passing `CSharpParseOptions` to the base ctor doesn't seem right.